ensure_command_output.sh

```diff
@@ -1,59 +1,43 @@
 #!/bin/bash
 
-MAGIC_END_STRING="END OF SUBCOMMAND"
 OUTPUT_SLEEP=60
 
 function run_command {
-  PIPE=$1
-  shift
   CMD="$@"
-  $CMD 1>$PIPE 2>$PIPE
-  echo "$MAGIC_END_STRING" 1>$PIPE
+  $CMD 
 }
 
 function run_ping {
```

Comment:
I would rather call it `print_with_interval`, and accept two parameters. `message` and `interval`